librz/bin/p/bin_z64.c

```diff
@@ -79,14 +78,36 @@ static bool check_buffer(RzBuffer *b) {
 	return !memcmp(magic, "\x80\x37\x12\x40", 4);
 }
 
+static bool n64_read_firm_hdr(RzBuffer *buf, N64Header *hdr) {
```

Comment:
Rename this also from `n64_read_firm_hdr` to `n64_read_hdr` since this is not a firmware.